packages/bot-strategy-runner/ExampleConfig.json

```diff
@@ -0,0 +1,14 @@
+{
+  "globalAddressWhitelistUrls": [
+    "https://raw.githubusercontent.com/UMAprotocol/protocol/master/packages/affiliates/payouts/devmining-status.json"
+  ],
+  "globalAddressBlacklist": [
+    "0x516f595978D87B67401DaB7AfD8555c3d28a3af4",
```

Comment:
What are these addresses? Would be helpful as an example to explain whether they are EMP's or something else

Reply:
These ones are either expired or we dont have price feeds for them. There are some contracts we run dev mining on that we dont run feeds on. that's what these addresses are for.

How do you recommend I add comments? json does not support comments.